services/compute/virtual-machines/taxonomy.md

```diff
@@ -0,0 +1,47 @@
+# Virtual Machines Taxonomy
+
+This _service-level taxonomy_ documents the minimal set of features
+that should be present for a service to be considered portable for
+use in financial services ecosystems.
+
+## Taxonomy
+
+| Taxonomy ID | Feature | Description |
+| ----------- | ------- | ----------- |
+|	CCC-030101	|	Instance Types	|	Offers a range of instance types with different specifications for CPU, GPU, memory, storage optimisation and networking capacity.	|
+|	CCC-030102	|	Operating System Options 	|	A selection of operating systems for virtual machine instances.	|
+|	CCC-030103	|	Ephemeral Storage	|	Temporary storage available to the VM which is lost when the instance is stopped or terminated	|
```

Comment:
What about the ability to plug in Permanent Storages. Like EBS with (with delete with instance option turned off) and EFS network drives in AWS.

Reply:
Is that covered by **Interoperability with Storage Options** below?